apps/molgenis-components/src/components/filters/DateFilter.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <InputRangeDate :id="id" :value="condition" @input="onUpdateCondition">
+    <template v-slot:append>
+      <button
+        v-if="condition"
+        @click="$emit('clearCondition')"
+        class="btn btn-outline-primary"
+        type="button"
+      >
+        <i class="fas fa-fw fa-times"></i>
+      </button>
+      <button
+        v-if="showAddButton && condition"
+        @click="$emit('addCondition')"
+        class="btn btn-outline-primary"
+        type="button"
+      >
+        <i class="fas fa-fw fa-plus"></i>
+      </button>
```

Comment:
I find it a bit sad that this code is not the outer component. I'd expect the the input to be slotted into this instead of the other way around. That way we'd also not need to be code for this inside of the input component.

Reply:
note use what you mean here , which comp are you referring when you say outer component. maybe we could change things around , but would suggest to have that as a separate PR. the input group was added after the rest of the code , so maybe when we have al the stuff we could flip it around, although not use if each type of component has the same append ( or prepend)